SU-SOE/stanford_soe_helper
# SOE-2234: Add .iframe-auto to allow body height to dynamically adjust…

# READY FOR REVIEW

# Summary
- This adds a the class .iframe-auto to allows text areas on the magazine article to adjust height for an iframe, such as audio.


# Needed By (Date)
- This is something the client is eagerly awaiting.

# Criticality
- Needed to display audio properly

# Steps to Test
- Pull this branch to a dev site
- Checkout PR#21 branch from stanford_magazine and follow steps to test from there

- Verify that the height is appropriate for images, video, and audio 
-  To verify audio, visit  magazine/article/enge-how-safe-and-secure-gps and add _iframe-auto_ class:
`<iframe class="iframe-auto" width="100%" height="166" scrolling="no" frameborder="no" src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/340021795&amp;color=%23ff5500&amp;auto_play=false&amp;hide_related=false&amp;show_comments=true&amp;show_user=true&amp;show_reposts=false"></iframe>`

# Affects 
- SoE

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2234
https://stanfordits.atlassian.net/browse/SOE-2296
https://stanfordits.atlassian.net/browse/SOE-2231
## Related PRs
https://github.com/SU-SWS/stanford_magazine/pull/21

## More Information
Before:
![screen shot 2017-10-11 at 10 05 05](https://user-images.githubusercontent.com/284440/31454975-0e135ca6-ae6c-11e7-964c-a53afc27f259.png)

After:
![screen shot 2017-10-11 at 10 05 17](https://user-images.githubusercontent.com/284440/31454980-143312ac-ae6c-11e7-8577-cf709323ade7.png)

- A temporary injector file has been added to _Prod_ to apply these changes until this PR has been approved and put on _Prod_. Once this is on _Prod_, delete the injector file.
https://engineering.stanford.edu/admin/config/development/css-injector/edit/37

## Folks to notify
@kbrownell 


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)… for audio
